src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence.mdx

```diff
@@ -162,78 +164,63 @@ These events are also available for querying, creating custom dashboards, and al
+  To use New Relic's anomaly detection, ensure you have:
 
-For more on data limits, see [Data limits](#data-limits).
+  * An APM agent installed for at least one application.
+  * To receive notifications in Slack, you'll need to ask your IT administrator to install the New Relic application in [your Slack workspace](https://slack.com/help/articles/202035138-Add-an-app-to-your-workspace).
 
-### Set up notifications for anomaly detection [#set-up]
+  For more on data limits, see [Data limits](#data-limits).
+  </Collapser>
 
-Automatic anomaly detection is enabled automatically, at no additional cost. To receive notifications or to have a configuration (group of apps) that you can add as a source for incident intelligence, you will need to create a anomaly detection configuration. 
+    <Collapser
+    className="freq-link"
+    id="jira"
+    title="Notifications"
+  >
+  Anomaly detection is enabled automatically, at no additional cost. To receive notifications or to have a configuration (group of apps) that you can add as a source for incident intelligence, you will need to create a anomaly detection configuration. 
 
-You can create a configuration in the anomaly detection UI:
+  You can create a configuration in the anomaly detection UI:
 
-1. From **[one.newrelic.com](https://one.newrelic.com/all-capabilities)**, click **Alerts & AI**.
-2. Under **Anomaly detection**, click **Settings**.
-3. Click **Add a configuration**.
-4. Input the following information into the form:
+  1. From **[one.newrelic.com](https://one.newrelic.com/all-capabilities)**, click **Alerts & AI**.
+  2. Under **Anomaly detection**, click **Settings**.
+  3. Click **Add a configuration**.
+  4. Input the following information into the form:
 
    * Choose a name for your configuration that helps you easily identify it from others in your account.
    * Select an account.
    * Select up to 1,000 applications. Note that certain applications with low throughput might not be good candidates for anomaly detection, as they can be more sensitive to smaller amounts of data fluctuation.
-5. Optional: select the golden signals you'd like to monitor for anomalies.
-6. Optional: [connect to incident intelligence](#add-source).
+  5. Optional: select the golden signals you'd like to monitor for anomalies.
+  6. Optional: [connect to incident intelligence](#add-source).
 
-<Callout variant="tip">
+  <Callout variant="tip">
 
-Workflows will soon be replacing configurations. Learn more about our workflows [here](/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows/).
+  Workflows will soon be replacing configurations. Learn more about our workflows [here](/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows/).
```

Comment:
I think you could eliminate the passive voice here: Workflows will soon replace configurations.